EX02/app/src/main/java/com/example/Game_of_throne/Stark1.java

```diff
@@ -0,0 +1,46 @@
+package com.example.Game_of_throne;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.media.MediaPlayer;
+import android.os.Bundle;
+import java.util.List;
+import java.util.ArrayList;
+import com.example.ex02.R;
+
+import static com.example.ex02.R.layout.lanister;
+import static com.example.ex02.R.layout.stark;
+
+public class Stark1 extends AppCompatActivity {
+
+    MediaPlayer mySongS;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(stark);
+        mySongS = MediaPlayer.create(this, R.raw.stark);
```

Comment:
Great idea of adding the media player and background sounds.
For the next time, think about a way to implement this once and not duplicating code - for example, maybe at the RecyclerViewAdapter?